File: util/excel/src/main/java/cn/bestwu/util/excel/ExcelCellSetter.java
package cn.bestwu.util.excel;

/**
 * 单元格值设置实体属性
 *
 * @param <T> 实体对象
 * @param <P> 属性
 */
@FunctionalInterface
public interface ExcelCellSetter<T, P> {

  void set(T entity, P property);
}

File: tools/excel/src/main/kotlin/top/bettercode/summer/tools/excel/ExcelCellSetter.kt
package top.bettercode.summer.tools.excel

import java.io.Serializable

/**
 * 单元格值设置实体属性
 *
 * @param <T> 实体对象
 * @param <P> 属性
</P></T> */
@FunctionalInterface
fun interface ExcelCellSetter<T, P> : Serializable {

    operator fun set(entity: T, property: P?)

}